src/middy.d.ts

```diff
@@ -0,0 +1,36 @@
+import {Callback, Context, Handler, ProxyResult} from 'aws-lambda';
+
+export interface IMiddy {
+  use: IMiddyUseFunction;
+  before: IMiddyMiddlewareFunction;
+  after: IMiddyMiddlewareFunction;
+  onError: IMiddyMiddlewareFunction;
+  __middlewares: object;
```

Comment:
I am not sure i would expose this one as it's supposed to be private (only exposed for testing purposes)

Reply:
Done